src/tests/test_utils/common_test_utils/src/graph_comparator.cpp

```diff
@@ -491,10 +493,20 @@ class CompareSubGraphs {
             }
         }
 
+        const auto lhs_body = sub_lhs->get_function();
+        const auto rhs_body = sub_rhs->get_function();
+        if (lhs_body && rhs_body) {
```

Comment:
Should we throw exception if one of bodies at least is `nullptr`?

Reply:
If both have empty bodies then these are equal. But if only one then there's a mismatch and invalid result should be return. I'll update it.